src/sections/Resume/Resume.jsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState } from "react";
+import { styles } from "../../styles";
+import {
+  SectionTitle,
+  SectionSubtitle,
+  TimelineMaker,
+  SkillMeter
+} from "./components";
+import { makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles(theme => ({
+  resumeSection: {
+    boxSizing: "border-box",
+    height: styles.sizes.globalDimensions.percentageFull,
+    overflowY: "scroll",
+    background: styles.colors.mediumGray,
+    marginLeft: styles.sizes.sidebarDimensions.width,
```

Comment:
If we are going to use this width for different sections then change this name to something more generic. Please apply this for any similar cases

Reply:
You can even use it as global dimensions